examples/maven-example/README.md

```diff
@@ -69,9 +69,9 @@ A Maven wrapper is included in this project and references a project specific se
 
 By using the maven wrapper, you will be able to build the project without having to install Maven locally.
 
-Linux/Mac: `./mvnw -s ./settings.xml clean install`
+Linux/Mac: `mvn -s ./settings.xml clean install`
 
-Windows: `mvnw -s .\settings.xml clean install`
+Windows: `mvn -s .\settings.xml clean install`
```

Comment:
- If you really want to remove the wrapper from the examples, address all of the work outlined in the first comment associated with this file.
- If you are okay with leaving the wrapper in with the example, accept the below suggestion.
```suggestion
Windows: `mvnw -s .\settings.xml clean install`
```